berkeley-mobile/StudyPact/Profile/ProfileViewController.swift

```diff
@@ -38,15 +38,13 @@ class ProfileViewController: UIViewController, UITextFieldDelegate, UINavigation
         
         setupHeader()
         
-        GIDSignIn.sharedInstance()?.delegate = self
-        let previousSignIn = GIDSignIn.sharedInstance().hasPreviousSignIn()
+        SignInManager.shared.addDelegate(delegate: self)
         
-        if let cryptoHash = StudyPact.shared.getCryptoHash(), previousSignIn {
-            // TODO: call GetUser and fill in profile
-            GIDSignIn.sharedInstance()?.restorePreviousSignIn()
+        if SignInManager.shared.isSignedIn,
+           let cryptohash = StudyPact.shared.getCryptoHash() {
```

Comment:
Isn't the `let cryptohash = StudyPact.shared.getCryptoHash()` line redundant since `.isSignedIn` checks all that jazz?

Reply:
this was just to unwrap the optional. could force unwrap I guess but I think this is better

Reply:
this was to unwrap it because the cryptohash was gonna be used here, but I ended up removing the thing that was gonna use the cryptohash

Reply:
removed this now